docs/contributing/development-environment.md

```diff
@@ -138,7 +138,7 @@ If you need to debug an issue that is occurring through use in matrix,
 say the unban command has stopped working, you can launch
 mjolnir from the JavaScript Debug Terminal using `yarn test:manual`.
 This will launch mjolnir using the config found in `config/harness.yaml`.
-You can now open https://app.element.io, change the server to `localhost:8081`,
+You can now open [https://app.element.io](https://app.element.io), change the server to `localhost:8081`,
```

Comment:
Quick way to trigger a build failure. I did exactly this before and it triggers a build failure.

Reply:
`<link>` isn't valid commonmark last I checked.

Reply:
The markdown linter thought it was fine but Docusaurus wont think that and will trigger a build failure.

Reply:
> `<link>` isn't valid commonmark last I checked.

It's valid commonmark. Search "autolink" in spec. Iirc it's even only part of commonmark and not others.

https://spec.commonmark.org/0.31.2/#autolinks

Reply:
> The markdown linter thought it was fine but Docusaurus wont think that and will trigger a build failure.

Smells like they parse mdx rather than commonmark markdown 👀

Reply:
> By default, Docusaurus v3 uses the MDX format for all files 

https://docusaurus.io/docs/markdown-features